### PR TITLE
Fix govuk-frontend assets

### PR DESCRIPTION
**Pull request by lfdebrux**
Fixes two issues with GOV.UK Frontend assets in this app:

1. Assets not being copied due to typo in gulpfile
2. Favicon not working in #1159

Related to https://trello.com/c/NjfXXwRH/1382-admin-frontend-has-no-favicon

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const npmRoot = path.join(repoRoot, 'node_modules')
 const govukCountryPickerDist = path.join(npmRoot, 'govuk-country-and-territory-autocomplete', 'dist')
 const govukToolkitRoot = path.join(npmRoot, 'govuk_frontend_toolkit')
 const govukElementsRoot = path.join(npmRoot, 'govuk-elements-sass')
-const govukFrontendRoot = path.join(npmRoot, 'digitalmarketplace-govuk-fronted', 'govuk-frontend')
+const govukFrontendRoot = path.join(npmRoot, 'digitalmarketplace-govuk-frontend', 'govuk-frontend')
```
Comment (katstevens) [inline gulpfile.js]: 🤦‍♀

**Comment by katstevens** [comment]
I'm happy to add these to my PR - save a functional test run?

**Comment by lfdebrux** [comment]
> I'm happy to add these to my PR - save a functional test run?

Sure, if that works for you!

File: gulpfile.js (hunk 1 above)
Comment (lfdebrux) [reply gulpfile.js]: Yeah, I did bad typing...

**Comment by lfdebrux** [comment]
Closing since @katstevens has incorporated these changes into #1159

**Pull request closed without merging**